src/track/cueinfo.cpp

```diff
@@ -0,0 +1,82 @@
+#include "track/cueinfo.h"
+
+#include "util/assert.h"
+
+namespace {
+const QString kDefaultLabel = ""; // empty string, not null
+}
+
+CueInfo::CueInfo()
+        : m_type(CueInfo::Type::Invalid),
+          m_startPositionMilliseconds(CueInfo::kNoPosition),
+          m_endPositionMilliseconds(CueInfo::kNoPosition),
+          m_hotCueNumber(CueInfo::kNoHotCue),
+          m_label(kDefaultLabel),
+          m_color(Color::kPredefinedColorsSet.noColor) {
+    DEBUG_ASSERT(!m_label.isNull());
+}
+
+CueInfo::CueInfo(
+        CueInfo::Type type,
+        double startPositionMilliseconds,
```

Comment:
Millis

Reply:
or maybe `Msecs`?

Reply:
"Millis" is very common and not unambiguious, i.e. **M**illis vs. **M**icros